packages/matrix-client-server/src/rooms/room_information/room_tags.ts

```
@@ -0,0 +1,113 @@
+import type MatrixClientServer from '../../'
+import {
+  send,
+  type expressAppHandler,
+  jsonContent,
+  validateParameters
+} from '@twake/utils'
+import { type Request } from 'express'
+
+export const getUserRoomTags = (
+  clientServer: MatrixClientServer
+): expressAppHandler => {
+  return (req, res) => {
+    const userId = (req as Request).params.userId
+    const roomId = (req as Request).params.roomId
+
+    clientServer.authenticate(req, res, (data, id) => {
+      clientServer.matrixDb
+        .get('room_tags', ['tag', 'content'], {
+          user_id: userId,
+          room_id: roomId
+        })
+        .then((tagRows) => {
+          const _tags: Record<string, { order?: number }> = {}
+          tagRows.forEach((row) => {
+            try {
+              const content = JSON.parse(row.content as string)
+              /* istanbul ignore else */
+              if (content.order !== undefined) {
+                _tags[row.tag as string] = { order: content.order }
+              } else {
+                _tags[row.tag as string] = {}
+              }
+            } catch (error) {
+              /* istanbul ignore next */
+              clientServer.logger.error(
+                'Error parsing room tag content:',
+                error
```

So errors are ignored when sending response. Is it the behavior we want ?